config/locales/email_subscribers.de.yml

```diff
@@ -0,0 +1,8 @@
+de:
```

Comment:
LGTM

Reply:
Thanks @julianrubisch. I added a bunch more translations just now if you're feeling keen 😅🙏

Reply:
There’s a general question to be asked here: do you want to keep the formal address („Sie“) or the informal one („du“)?

Reply:
Informal is the goal! It's how it is in English ("Hi" vs "Hello" etc).

Reply:
okay, then I have more to do 😅